templates/web/base/waste/garden/subscribe_summary.html

```diff
@@ -2,16 +2,24 @@
 title = 'Subscribe to green garden waste collections';
 thing = 'garden subscription';
 summary_title = 'Subscription';
-IF form.saved_data.container_choice == 'sack';
-    step1 = 'sacks_details';
+IF form_class.match('Renew');
+    IF form.saved_data.container_choice == 'sack';
+        step1 = 'sacks_choice';
+    ELSE;
+        step1 = 'intro';
+    END;
 ELSE;
-    step1 = 'details';
+    IF form.saved_data.container_choice == 'sack';
+        step1 = 'sacks_details';
+    ELSE;
+        step1 = 'details';
+    END;
 END;
 %]
 
 [% BLOCK answers %]
     <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key govuk-summary-list__key--sub">Green garden waste subscription</dt>
+        <dt class="govuk-summary-list__key govuk-summary-list__key--sub">Green garden waste collection</dt>
```

Comment:
The word change is because that's what it said in the (now gone) renew template and I wanted to merge the two and that seemed like the better wording. The different step1 is because the subscription flow has a static "intro" page before the bit you'd like to edit, whereas the renew flow does not.